api/routes/system.py

```
@@ -0,0 +1,1224 @@
+            )
+        
+        # Get logs using the log reader service
+        log_data = get_logs_by_category(category, lines)
+        
+        return {
+            "status": "success",
+            "data": log_data
+        }
+        
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)